tortoisebeacon/tortoise_beacon.go

```diff
@@ -613,25 +608,27 @@ func (tb *TortoiseBeacon) runConsensusPhase(ctx context.Context, epoch types.Epo
 	ticker := time.NewTicker(tb.votingRoundDuration + tb.weakCoinRoundDuration)
 	defer ticker.Stop()
 
-	tb.sendFollowingVotesLoopIteration(ctx, epoch, firstRound+1)
+	for round := firstRound; round <= tb.lastPossibleRound(); round++ {
+		go tb.startWeakCoin(ctx, epoch, round)
```

Comment:
We've already discussed the change here, but I'm leaving a comment not to forget about it.

Reply:
what are you referring to? i wasn't planning to make other changes here

Reply:
Sorry, I didn't check the body of `tb.startWeakCoin` before leaving this comment and thought that you start weak coin before voting. What do you think of putting `go tb.startWeakCoin(ctx, epoch, round)` below the goroutine below to show that its actual code is run after sending votes? Just for readability